.github/workflows/quality-assurance.yml

```
@@ -0,0 +1,671 @@
+        with:
+          name: load-test-results
+          path: |
+            load-test-results.json
+            load-test-report.html
+          retention-days: 30
+
+  # Accessibility testing
+  accessibility-testing:
+    name: Accessibility Testing
+    runs-on: ubuntu-latest
+    if: hashFiles('**/*.html', '**/public/**/*') != ''
```

actionlint (1.7.7)

calling function "hashFiles" is not allowed here. "hashFiles" is only available in "jobs.<job_id>.steps.continue-on-error", "jobs.<job_id>.steps.env", "jobs.<job_id>.steps.if", "jobs.<job_id>.steps.name", "jobs.<job_id>.steps.run", "jobs.<job_id>.steps.timeout-minutes", "jobs.<job_id>.steps.with", "jobs.<job_id>.steps.working-directory". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)